### PR TITLE
Added scripts to get kafka topics

### PR DESCRIPTION
**Pull request by robshelly**
Added a script that runs the `kafka-topics.sh` scripts that is built into Kafka pods. It's used to get details of Kafka topics and partitions and also filter by under replicated, under min ISR or offline partitions.

### CODE REVIEW

**Comment by Nanyte25** [comment]
@wanghaoran1988 can you review this pr

**Comment by reetika-vyas** [comment]
The build is  failed with below error 

```
16:12:38 Complete!
16:12:39 
16:12:39 In ./CSSRE/get-kafka-topics/get-kafka-topics.sh line 14:
16:12:39     echo $usage
16:12:39          ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39 
16:12:39 
16:12:39 In ./CSSRE/get-kafka-topics/get-kafka-topics.sh line 22:
16:12:39 if [ -z ${kafka_namespace} ]; then
16:12:39         ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39 
16:12:39 
16:12:39 In ./CSSRE/get-kafka-topics/get-kafka-topics.sh line 41:
16:12:39   kafka_cluster=$(oc -n ${kafka_namespace} get kafka --no-headers | awk '{print $1}')
16:12:39                         ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39 
16:12:39 
16:12:39 In ./CSSRE/get-kafka-topics/get-kafka-topics.sh line 44:
16:12:39   if [ -z ${topic} ]; then
16:12:39           ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39 
16:12:39 
16:12:39 In ./CSSRE/get-kafka-topics/get-kafka-topics.sh line 45:
16:12:39     oc -n ${kafka_namespace} exec -it statefulset/${kafka_cluster}-kafka -c kafka -- env - bin/kafka-topics.sh --bootstrap-server localhost:9096 --describe ${filter}
16:12:39           ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39                                                   ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39                                                                                                                                                             ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39 
16:12:39 
16:12:39 In ./CSSRE/get-kafka-topics/get-kafka-topics.sh line 47:
16:12:39     oc -n ${kafka_namespace} exec -it statefulset/${kafka_cluster}-kafka -c kafka -- env - bin/kafka-topics.sh --bootstrap-server localhost:9096 --describe --topic ${topic} ${filter}
16:12:39           ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39                                                   ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39                                                                                                                                                                     ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39                                                                                                                                                                              ^-- SC2086: Double quote to prevent globbing and word splitting.
16:12:39 
16:12:40 make: *** [shellcheck] Error 123
16:12:40 Build step 'Execute shell' marked build as failure
16:12:40 Setting status of b0bd9be637eab29149d962b0929e1c6a9dfff936 to FAILURE with url https://ci.ext.devshift.net/job/openshift-managed-scripts-gh-pr-check/267/ and message: ' '
16:12:40 Using context: ci.ext.devshift.net PR build
16:12:40 Finished: FAILURE



```

File: scripts/CSSRE/get-kafka-topics/get-kafka-topics.sh [hunk 1]
```diff
@@ -11,15 +11,15 @@ while getopts ":n:t:f:" opt; do
     f) filter="${OPTARG}"
     ;;
     \?)
-    echo $usage
+    echo "$usage"
```
Comment (georgettica) [inline scripts/CSSRE/get-kafka-topics/get-kafka-topics.sh:14]: FYI, you can also do "${usage}" 

relevant to all changes

**Comment by Nanyte25** [comment]
@robshelly and @georgettica and @wanghaoran1988 I have tested this script locally and on cluster 
```
./get-kafka-topics.sh -n kafka-c6rm80r0ubtldafuhvhg -t test-topic
Namespace: kafka-c6rm80r0ubtldafuhvhg
Topic: test-topic
Filter: --

Getting kafka topics
Kafka cluster: mk-e2e-ll-e2e-stage
Topic: test-topic        TopicId: GPI0h3LKT_CCWt4RbBJwZQ        PartitionCount: 1        ReplicationFactor: 3        Configs: min.insync.replicas=2,message.format.version=3.0-IV1,max.message.bytes=1048588
        Topic: test-topic        Partition: 0        Leader: 2        Replicas: 2,0,1        Isr: 1,2,0


```

**Pull request merged**